js/pricing.js

```diff
@@ -7,27 +10,145 @@
 
 $(
     function () {
-        const $confirmPersonalInformation = $('#confirm-personal-information');
-        const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $privacyConsent = $('#privacyConsent');
+        const $supportAgreementConsent = $('#supportAgreementConsent');
+        const $orderButton = $('#order-now-btn');
+        const $redirectScreen = $('#redirectScreen');
+        const $loader = $redirectScreen.find('.redirect-loader');
+        const $errorContainer = $redirectScreen.find('.redirect-error');
+        const $linkBack = $redirectScreen.find('#linkBack');
+        const $consentCheckboxes = $("input[type='checkbox'].consent");
 
-        $confirmPersonalInformation.change(function () {
-            confirmAgreement(this, $disabledButton);
+        const orderUrl = "{{site.data.payment_config.orderUrl}}";
+
+        let apiUrl;
+        if ("{{jekyll.environment}}" === "development") {
+            apiUrl = "{{site.data.payment_config.devApiUrl}}";
+        } else {
+            apiUrl = "{{site.data.payment_config.prodApiUrl}}";
+        }
+
+        $consentCheckboxes.change(() => {
+            changeElementState($orderButton, isConsentObtained());
+        });
+
+        $orderButton.click(() => {
+            if (isConsentObtained()) {
+                submitOrder();
+            }
         });
 
-        $confirmDevelopmentAgreement.change(function () {
-            confirmAgreement(this, $disabledButton);
+        $linkBack.click(e => {
+            e.preventDefault();
+            hideRedirect();
         });
 
-        function confirmAgreement(element, disabledElement) {
-            const isConfirmed = $confirmPersonalInformation.prop('checked') && $confirmDevelopmentAgreement.prop('checked');
-            if (isConfirmed) {
-                disabledElement.removeAttr('disabled');
-                disabledElement.removeClass('disabled');
+        /**
+         * Checks if all consent checkboxes are checked.
+         *
+         * @return {boolean} true if all checkboxes are checked
+         */
+        function isConsentObtained () {
```

Comment:
Please make sure you format the code before submitting it. Also, make sure to have the auto-format checkbox checked in the IDE commit dialog.